src/config.rs

```diff
@@ -75,6 +77,10 @@ pub struct Config {
     /// Note: math.MaxUusize64 for unlimited, 0 for at most one entry per message.
     pub max_size_per_msg: u64,
 
+    /// Limit the size of the committed entries which can be applied.
+    /// Note: Use math.MaxUusize64 for unlimited, 0 for at most one entry per message
```

Comment:
`math.MaxUusize64` is invalid in rust. I think NO_LIMIT is a public constant in raft crate?

Reply:
Could I just use `raft::{ NO_LIMIT}` instead of `math.MaxUsize64` in the comments above? ( I'm not familiar with rust comment tricks )